.github/workflows/release.yaml

```
@@ -0,0 +1,45 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - release/*
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    environment: Release
+    permissions:
+      contents: write
+    steps:
+      - uses: actions/checkout@v2
+      - name: Read version to release from the changelog
+        id: next-release
+        uses: cucumber-actions/changelog-action@v1.3
+        with:
+          args: latest
+      - name: Set version environment variable
+        run: echo "version"=${{ steps.next-release.outputs.result }} > $GITHUB_ENV
+      - name: read latest version from the changelog
+        id: release-notes
+        uses: mattwynne/changelog-action@v1.3
+        with:
+          args: show ${{ env.version }}
+      - name: Create release
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          cat >${{ runner.temp }}/notes <<EOT
+          ${{ steps.release-notes.outputs.result }}
+          EOT
+          gh release create \
+            --notes-file ${{ runner.temp }}/notes \
+            v${{ env.version }}
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 2.6
+          bundler-cache: true
+      - name: Release to Rubygems
+        run: |
+          bundle exec gem build
+          bundle exec gem push cucumber-wire-${{ env.version }}.gem -k ${{ env.CUKEBOT_GITHUB_TOKEN }}
```

```suggestion
          bundle exec gem push cucumber-wire-${{ env.version }}.gem -k ${{ secrets.RUBYGEMS_API_KEY }}
```
Shouldn't we use the RUBYGEMS_API_KEY here?

---

Yes, and in fact we should put it in an environment variable. See afcd5c9